webpack.config.umd.js

```diff
@@ -10,26 +8,30 @@ module.exports = {
     filename: "index.js",
     libraryTarget: "umd"
   },
+  resolve: {
+    extensions: [".js", ".jsx"]
+  },
   module: {
     rules: [
       {
         loader: "babel-loader",
         test: /\.(js|jsx)$/,
         exclude: /node_modules\/proptypes|scripts\/sw.js/,
         options: {
-          presets: [["es2015", { modules: false }]],
+          babelrc: false,
```

Comment:
I agree that this is ugly, maybe there's a more elegant way to do this by just adding the es2015 preset onto babelrc's config...?